bower.json

```diff
@@ -19,6 +19,7 @@
     "locales": "https://github.com/citation-style-language/locales.git#d2b612f8a6f764cbd66e67238636fac3888a7736",
     "raven-js": "2.1.0",
     "zeroclipboard": "2.1.6",
-    "osf-style": "https://github.com/CenterForOpenScience/osf-style.git#5dcf5997ad2808ffc7c8564f94cde102bf7a4a96"
+    "osf-style": "https://github.com/CenterForOpenScience/osf-style.git#5dcf5997ad2808ffc7c8564f94cde102bf7a4a96",
+    "At.js": "https://github.com/CenterForOpenScience/At.js.git#d4c1c6ebce3808a275bac59740dabd9bcf76d96f"
```

Comment:
Did we consider changing the behavior through an override (cf. https://github.com/CenterForOpenScience/osf.io/pull/5840#discussion_r67533780) rather than a complete fork?